docs/supported_languages_and_lockfiles.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: Supported Languages and Lockfiles
+permalink: /supported-languages-and-lockfiles/
+nav_order: 2
+---
+
+# Supported Languages and Lockfiles
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+## Supported lockfiles
+
+A wide range of lockfiles are supported by utilizing this [lockfile package](https://github.com/google/osv-scanner/tree/main/pkg/lockfile).
+
+|  Language  | Compatible Lockfile(s) |
+|:-----------|:------------------|
+| Elixir | `mix.lock` |
+| C/C++ | `conan.lock`|
+| Go | `go.mod` |
+| Java | `buildscript-gradle.lockfile`<br>`gradle.lockfile`<br>`pom.xml`[\*](https://github.com/google/osv-scanner/issues/35)<br>`yarn.lock` |
+| Javascript | `package-lock.json`<br>`packages.lock.json`|
+| PHP        | `composer.lock`|
+| Python     | `Pipfile.lock`<br>`poetry.lock`<br>`requirements.txt`[\*](https://github.com/google/osv-scanner/issues/34) |
+| Ruby       | `Gemfile.lock`|
+| Rust       | `Cargo.lock`|
+
+## Alpine Package Keeper and Debian Package Keeper
+
+The scanner also supports:
+- `installed` files used by the Alpine Package Keeper (apk) that typically live at `/lib/apk/db/installed`
+- `status` files used by the Debian Package manager (dpkg) that typically live at `/var/lib/dpkg/status`
+
+however you must [specify](./usage.md/#specify-lockfiles) them explicitly using the `--lockfile` flag:
+
+```bash
+osv-scanner --lockfile 'apk-installed:/lib/apk/db/installed'
+osv-scanner --lockfile 'dpkg-status:/var/lib/dpkg/status'
+```
+
+## C/C++ scanning
+
+OSV-Scanner also supports vendored and submoduled C/C++ dependencies. 
+
+Because the C/C++ ecosystem does not have a centralized package manager, C/C++ dependencies tend to be bundled with the project. Dependencies are either [submoduled](#submoduled-dependencies) or [vendored](#vendored-dependencies). In either case, OSV-Scanner is able to find known vulnerabilities in your project dependencies. 
+
+OSV-Scanner's C/C++ support is based on commit-level data. OSV's commit-level data covers the majority of C/C++ vulnerabilities within the OSV database, but users should be aware that there may be vulnerabilities in their dependencies that could be excluded from OSV-Scanner results. Adding more commit-level data to the database is an ongoing project. 
```

Comment:
Can we add a reference to "vulnerable commit ranges" here? That's the terminology we used in the blog post.